contracts/javascore/ibc/src/main/java/ibc/ics25/handler/IBCHandlerChannel.java

```diff
@@ -55,6 +55,7 @@ public void channelOpenInit(MsgChannelOpenInit msg) {
                 channel.getCounterparty().encode(),
                 channel.getVersion());
         claimCapability(channelCapabilityPath(msg.getPortId(), id), module._address());
+        addPortId(msg.getPortId());
```

Comment:
so, we want to add the port in this method itself rather than adding separately ?

Reply:
i think @izyak  knows more on this.

Reply:
We added this while testing. The changes equivalent to this have already been merged to main via another PR, and this is not necessary here. So, reverting the javascore changes